cypress/integration/login.js

```diff
@@ -1,70 +1,78 @@
+import LoginPage from '../objects/loginPage';
+
+const loginPage = new LoginPage();
+
 describe('Sourcebooks login', function() {
 
-    it('Should display validation for empty user after attempted loggin', function () {
+    it('Should display validation for empty user after attempted login', function () {
         
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
+        loginPage.visit();
+        loginPage.getUserValidationIndicator().should('not.visible');
+        loginPage.clickSubmitButton();
+        loginPage.getUserValidationIndicator().should('be.visible');
     })
 
     it('Should be able to login with role Team Lead', function () {
 
-        cy.visit('/');
-        cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Dominykas Poškus"]').click();
-        cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="Team Lead"]').click();
-        cy.get('[type="submit"]').click();
+        loginPage.visit();
+        loginPage.clickUserDropDown();
+        loginPage.clickSpecificUserFromDropDown("Dominykas Poškus");
+        loginPage.clickRoleDropDown();
+        loginPage.clickSpecificRoleFromDropDown("Team Lead");
+        loginPage.clickSubmitButton();
 
-        cy.url().should('include', '/time-logging');
-        cy.get('.page__title').contains('Timesheets');
-        cy.get('.calendar').should('be.visible');
-        cy.get('.tile.form').should('be.visible');
-        cy.get('.user-info__title').contains('Dominykas Poškus');
-        cy.get('.main-nav').find('li').should('have.length', 2);
+        loginPage.checkIfUrlIncludes('/time-logging');
+        loginPage.getPageTitle().contains("Timesheets");
+        loginPage.getCalendar().should('be.visible');

```

Comment:
Methods with time logging elements should be put in `timeLoggingPage.js`. Also elements that are shared between pages (like navigation bar) should be put in separate file like `common.js` or `pageComponents.js`